templates/coredns/0/rancher-compose.yml

```diff
@@ -0,0 +1,84 @@
+.catalog:
+  name: "CoreDNS"
+  version: "1.0.1"
+  minimum_rancher_version: v1.6.14
+  description: "CoreDNS is a DNS server that chains plugins and provides such DNS Services like Etcd & Kubernetes. For kubernetes, please use helm coredns's chart."
+  questions:
```

Comment:
What do you think if you add a question `PUBLISH_PORT`, with default value of 53, to be able to publish the service in configurable port??
```
+    - variable: PUBLISH_PORT
+      description: "Port to publish coredns service"
+      label: "Publish Port"
+      required: true
+      default: "53"
+      type: "int"
```